### PR TITLE
Add alpha trace and tview-template commands

### PR DESCRIPTION
**Pull request by sttts**
```bash
$ up alpha trace providers
```

Warning: this requires an Apollo API at http://localhost:8080/apis/spaces.upbound.io/v1alpha1/query. This will change when mxe-router learns about the Spaces APIs.

Bildschirmfoto 2024-02-14 um 14 40 51

`up alpha tview-template`:

image

### CODE REVIEW

File: cmd/up/trace/cmd.go [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+
+	spacesv1alpha1 "github.com/upbound/up/cmd/up/trace/spaces/v1alpha1"
+)
+
+type Cmd struct {
+	ControlPlane string `short:"c" long:"controlplane" env:"UPBOUND_CONTROLPLANE" description:"Controlplane to query"`
+	Group        string `short:"g" long:"group" env:"UPBOUND_GROUP" description:"Group to query"`
+	Namespace    string `short:"n" long:"namespace" env:"UPBOUND_NAMESPACE" description:"Namespace of objects to query"`
```
Comment (AlainRoy) [inline cmd/up/trace/cmd.go:36]: It's worth saying what the default are if these aren't specified.

Comment (sttts) [reply cmd/up/trace/cmd.go:50]: done

**Review by Piotr1215 — APPROVED** [review]
Thank you @sttts . It's too big scope for me to review everything, but specifically the `tview` looks great. It would be nice to have a dedicated short README on how to use it.

**Comment by sttts** [comment]
@Piotr1215 added doc.go which should give some background and links.

**Pull request merged**